.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+### Description
+<!-- Example: The `vaadin-checkbox` element does not get checked when clicked. -->
+
+### Live Demo
+<!-- Click "Remix This" and edit -- must be logged in to persist! -->
+https://glitch.com/edit/#!/vaadin-checkbox-issue
```

Comment:
This makes the issue reporting on this repo very Web Component -specific. Nowadays we also host the Flow component issues in the repo.

Reply:
I agree that we need proper issue templates but that could be done separately.